add customer deletion (#325)
meshcloud/meshcloud-docs: docs/meshcloud.customer.md

## Remove assigned meshCustomer Roles

If you would like to remove a user or group from your meshCustomer go to your "Account" settings and select "Customer Access". You can click the "trash" icon in the "Customer Access" section to remove the user or group from your meshCustomer. If 4-AP is active in your meshInstallation and the role request has not been approved by another Customer Admin yet, click the "trash" icon in the Pending Role Requests section. When removing someone from the meshCustomer, the user or group is automatically removed from all projects it had access to. All users won't be able to access cloud resources of your projects anymore, if they are not assigned via another role binding anymore. The users or members of the group will be informed via email, that their access to the meshCustomer was revoked.

## Customer Deletion

> meshCustomers can be deleted under certain conditions. Before a meshCustomer is deleted, a check is performed to verify the following:

Reviewer: ```suggestion
Before a meshCustomer may be deleted, a check is performed to verify the following:
```
